lib/hanami/action/mime.rb

```diff
@@ -41,6 +41,64 @@ module Mime
       # @api private
       DEFAULT_CHARSET = 'utf-8'.freeze
 
+      # Most commom mime types used for responses
+      #
+      # @since 0.9.0
+      # @api private
+      MIME_TYPES = {
```

Comment:
> Yeah, I tried to avoid the multiple file extension trouble

Of course, the problem of hanami/hanami#663 is caused by duplicates. If we introduce them again, we haven't solved too much 😉 .